play/src/pusher/services/AdminApi.ts

```diff
@@ -991,6 +1004,21 @@ class AdminApi implements AdminInterface {
         return;
     }
 
+    //TODO : @openapi doc
```

Comment:
Can we try not to forget this one?
People implementing an admin API will thank us.